Framework/DataHandling/src/SaveGSS.cpp

```diff
@@ -169,6 +211,9 @@ void SaveGSS::exec() {
     entry = makeStringStream();
   }
 
+  // process user special headers
+  processUserSpecifiedHeaders();
+
   // Check the user input
   validateUserInput();
```

Comment:
Change this to `validateInputs()` and remove from `exec()`. Then the framework will automatically call it at the right time.